config/routes.rb

```diff
@@ -10,21 +10,21 @@
     resource  :duplicate,         only: %i[new create],                 controller: 'grants/duplicate'
     resource  :state,             only: :update,                        controller: 'grants/state'
     member do
-      get   'criteria',        to: 'grants/criteria#index',  as: :criteria
-      patch 'criteria/update', to: 'grants/criteria#update', as: :update_criteria
+      get   'criteria',           to: 'grants/criteria#index',   as: :criteria
+      patch 'criteria/update',    to: 'grants/criteria#update',  as: :update_criteria
     end
     resources :forms,             only: %i[update edit update_fields],  controller: 'grant_submissions/forms' do
       member do
         put :update_fields
       end
     end
-
-    resources :submissions,       except: %[new show],       controller: 'grant_submissions/submissions'
+    resources :submissions,       except: %i[new],       controller: 'grant_submissions/submissions' do
+      resources :reviews,         except: %[index new],  controller: 'reviews', as: :submission_reviews
```

Comment:
why as: submission_reviews?

Reply:
removed submission_reviews named route and updated controllers/views.